drafts/build_a_dao_on_fuel/draft.md

```diff
@@ -0,0 +1,1459 @@
+[note]: <> ("note to self and file editor, there are specific comments left throughout this file, for the sake of understanding the author's reason for using a particular method of approach; pls feel free to delete once acknowledged or reply edit with a better approach 🥂")
+
+# How to Build a DAO on the Fuel Blockchain.
+
+## Introduction
+
+A Decentralized Autonomous Organization (DAO) is considered an essential blockchain application.
+
+It is a Decentralized application that users or organizations use to automate governance execution of their projects and communities
+It is an automated program represented by rules encoded in a computer that is transparently controlled by the organization members and not influenced by any central authority.
+
+On completing this article, you will learn how to build a full-stack DAO on the fuel blockchain; you will touch down on the following concepts of development;
```

Comment:
I've found the following issues: 
- Informal language: using "touch down on". Consider replacing it with "explore," "learn about," or similar expressions.
- Punctuation: using semicolon incorrectly. A colon would be more appropriate to introduce the list of concepts that will be covered.
- Capitalization: "Fuel" in "fuel blockchain" should be capitalized, as it is a proper noun referring to a specific blockchain.
- Style: using "On" in  "On completing this article" might be imprecise. A better option could be "By completing this article".
- Precision: using"concepts of development" is imprecise since the listed elements after this sentence are sections of a tutorial, rather than concepts of software development.  Please consider a phrase such as "This tutorial covers the following steps of dapp development".

Please consider using spell checkers in your code editor such as [LTEX](https://marketplace.visualstudio.com/items?itemName=valentjn.vscode-ltex).

Reply:
Thanks for the tips I'm now using LTEX
For this review 
- I've removed all the usage of informal speech from the article.
-  The piece is now Grammarly proof, and I've corrected the Fuel nouns (Capitalization) appropriately throughout the piece. This also resolves the proper use of punctuation in the whole piece.
- I've also used the recommended and appropriate style throughout the piece.
- I've also used your suggested all the precision recommendations you mentioned with a few more modifications.